ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -758,9 +778,21 @@ func resourceIBMContainerVpcClusterUpdate(d *schema.ResourceData, meta interface
 			return err
 		}
 
-		if err := ClusterClient.VPCs().SetOutboundTrafficProtection(clusterID, outbound_traffic_protection, Env); err != nil {
-			return err
+		if d.HasChange(DisableOutboundTrafficProtectionFlag) {
+			outbound_traffic_protection := !d.Get(DisableOutboundTrafficProtectionFlag).(bool)
+			if err := ClusterClient.VPCs().SetOutboundTrafficProtection(clusterID, outbound_traffic_protection, Env); err != nil {
+				return err
+			}
 		}
+
+		if d.HasChange(EnableSecureByDefaultFlag) {
```

Comment:
I'm not sure, I can follow you. 
This resource flag can be set to only true. Basically an allready created cluster where this flag was not set, it can be set to true, but after that it can't be set to any other value. The validation for the "only true" is where the flag is described: https://github.com/IBM-Cloud/terraform-provider-ibm/blob/ee589a7fe523c298e1f94f8113730990546b403f/ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go#L367